### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by jicsontoh**

### CODE REVIEW

**Review by Jo3LW — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -5,12 +5,10 @@ title: User Guide
 
 ## Introduction
 
-JeeqTracker is a desktop application created for **home-based business owners / resellers**. It provides a fast and simple
-way for you to keep track of **`BUY / SELL`** transactions with clients, and allows you to add **`REMARKS`** of clients.
-With JeeqTracker, you can quickly find out who you are earning the most profit from, who is causing you to lose money, and
-who is not a pleasant person to work with. This will help your business prosper and yield more profits.
+JeeqTracker is a desktop application created for **home-based business owners**. It helps 
+you keep track of your clients’ contact details, transaction records, and remarks about the client.
```
Comment (ElginL) [inline docs/UserGuide.md:9]: Should we add a value proposition here?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -52,7 +50,8 @@ To get you started, you might want to read the section on [how to use this User
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your Computer. If you do not have Java, you can download it from 
+[here for Window Users](https://www.oracle.com/java/technologies/downloads/#java11) and [here for Mac Users](https://www.azul.com/downloads/?version=java-11-lts&os=macos&architecture=arm-64-bit&package=jdk-fx).
```
Comment (ElginL) [inline docs/UserGuide.md:67]: Nice

**Review by ElginL — APPROVED** [review]
LGTM

**Pull request merged**